### PR TITLE
JOSS submission

### PR DESCRIPTION
**Pull request by jhollist**
Added materials for JOSS submission.  Also added some additional USEPA disclaimers.

Will need to pass through USEPA review which will possibly have some changes to vigentte or documentation.   Will submit those changes in next couple of weeks.

### CODE REVIEW

**Comment by sckott** [comment]
thanks @jhollist - so should we wait on this then?

**Comment by jhollist** [comment]
I should have a couple more commits coming.  Let's hold off until then to do the merge.

If you have any comments on the summary in paper.md let me know and I will make those changes as well.

Hopefully the bureaucratic wheels don't move too slowly.

File: paper.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+title: 'lawn: An R client for Turfjs for Geospatial Analysis'
```
Comment (sckott) [inline paper.md]: maybe change to 'Turf.js'

Comment (sckott) [inline paper.md]: i don't know, maybe it doesn’t matter

Comment (jhollist) [inline paper.md]: done!

Comment (jhollist) [inline paper.md]: maybe more verbose is better? 

"the Turf javascript library"

**Comment by jhollist** [comment]
@sckott I think this is ready to go now.  Have recieved internal comments back (may get a few more, but unlikely).  Still want to deal with #56 but don't want to make this PR too big.  Will do in a separate one.

**Comment by sckott** [comment]
cool, looks good.

**Pull request merged**